app/models/allocation.rb

```diff
@@ -25,5 +45,16 @@ class Allocation < VersionedModel
   validates :moves_count, presence: true, numericality: { only_integer: true, greater_than: 0 }
   validates :date, presence: true
 
+  # TODO: re-enable once cancellation reason is implemented in FE
```

Comment:
Cancellation reason and comment was de-scoped after I already implemented it, so I thought it might be better to keep here until the frontend team starts implementing it soon

Reply:
this change may break other FE features, which now require and expect a cancellation reason